src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int BOUND = 100;
+    private final Ball ball = new Ball();
```

Comment:
You should not create a single instance of Ball at the class level if you're going to return a different Ball object each time 'getRandomBall()' is called. This can lead to unexpected behavior since the same Ball object is being mutated and returned with each call. Instead, create a new Ball object within the 'getRandomBall()' method.